pyalex/api.py

```diff
@@ -401,6 +416,20 @@ class Funders(BaseOpenAlex):
     resource_class = Funder
 
 
+class Autocomplete(OpenAlexEntity):
+    pass
+
+
+class Autocompletes(BaseOpenAlex):
+    """ Class to autocomplete without being based on the type of entity """
+    resource_class = Autocomplete
+
+    def __getitem__(self, key):
+        return self._get_from_url(
+            config.openalex_url + "/autocomplete" + "?q=" + key, return_meta=False
+        )
```

Comment:
Ideally, we can make this into something like this:

```python
def autocomplete(q, return_meta=False):
    BaseOpenAlex().autocomplete(q, return_meta=return_meta)
```

I expect this to require some changes to the base class as well.